eng/readme-templates/README.md

```diff
@@ -58,6 +58,9 @@ The [.NET Framework Docker samples](https://github.com/microsoft/dotnet-framewor
 * [dotnet/samples](https://hub.docker.com/_/microsoft-dotnet-samples/): .NET Samples
 
 {{if !IS_PRODUCT_FAMILY:# Full Tag Listing
+<!--End of generated tags-->
+
+*Tags not listed in the table above should be considered unsupported.*
```

Comment:
What's the reason for capitalizing 'Listed'?
Consider using stronger language than "should be" - they are definitely unsupported.

Tags excluded from the above table are unsupported.  See the [Supported Tags Policy](https://github.com/microsoft/dotnet-framework-docker/blob/main/documentation/supported-tags.md).

Reply:
> What's the reason for capitalizing 'Listed'?

It's not. That's just the italic font.